error_test.go

```diff
@@ -300,24 +310,22 @@ var _ = Describe("Merge", func() {
 		})
 	})
 
-	Context("with a nil argument", func() {
-		const code = "foo"
+	Context("with a nil target", func() {
```

Comment:
nitpick: the test is calling `MergeErrors` so really it's still an argument and not a target?